plugins/filter/remove_keys_from_values.py

```diff
@@ -0,0 +1,213 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2024 Vladimir Botka <vbotka@gmail.com>
+# Copyright (c) 2024 Felix Fontein <felix@fontein.de>
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import annotations
+
+DOCUMENTATION = r"""
+name: remove_keys_from_values
+short_description: Remove keys from a list of dictionaries or a dictionary in base of value content
+version_added: "10.6.0"
+author:
+  - Lorenzo Tanganelli (@tanganellilore)
+description:
+  - This filter removes keys from a list of dictionaries or a dictionary,
+  - recursively or not depending on the parameters.
```

Comment:
If this is meant to be a single sentence, please do:
```suggestion
  - This filter removes keys from a list of dictionaries or a dictionary,
    recursively or not depending on the parameters.
```